File: Praktikum I Modul 1/Praktikum I Modul1 26-09-2023/C/PRAK104-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>

int main(){

    float hargaSepatuA = 400000;
    float hargaSepatuB = 350000;

    float diskonSepatuA = 0.13;
    float diskonSepatuB = 0.21;

    float hasil_potonganA = hargaSepatuA * diskonSepatuA;
    float hasil_potonganB = hargaSepatuB * diskonSepatuB;

    float hasilA = hargaSepatuA - hasil_potonganA;
    float hasilB = hargaSepatuB - hasil_potonganB;

    printf("Harga sepatu A adalah %.0f\n", hargaSepatuA);
    printf("Harga sepatu B adalah %.0f\n", hargaSepatuB);
    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %.0f\n", hasilA);
    printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi %.0f\n", hasilB);

    return 0;
}

File: Praktikum I Modul 1/Praktikum I Modul1 26-09-2023/Python/PRAK104-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
hargaSepatuA = 400000
hargaSepatuB = 350000

diskonSepatuA = 0.13
diskonSepatuB = 0.21

hasil_potonganA = hargaSepatuA * diskonSepatuA
hasil_potonganB = hargaSepatuB * diskonSepatuB

hasilA = int(hargaSepatuA - hasil_potonganA)
hasilB = int(hargaSepatuB - hasil_potonganB)

print("Harga sepatu adalah",hargaSepatuA)
print("Harga sepatu adalah",hargaSepatuB)
print("Sepatu mendapat diskon 13%","sehingga harganya menjadi",hasilA)
print("Sepatu mendapat diskon 21%","sehingga harganya menjadi",hasilB)